content/en/docs/alerts/_index.md

```diff
@@ -178,3 +178,62 @@ Here's the same output, pretty-printed:
 ## gRPC Output
 
 If you'd like to send alerts to an external program connected via gRPC API (for example, the [falco-exporter](https://github.com/falcosecurity/falco-exporter)), you need to enable both the `grpc` and `grpc_output` options as described under the [gRPC Configuration section](/docs/grpc/#configuration).
+
+
+{{% pageinfo color="primary" %}}
+
+_SELinux note_: If you have [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux) enabled you may see the following error while starting Falco with gRPC support enabled:
+
+```shell
+
+May 08 12:14:41 myserver falco[15442]: Falco version X.XX.X (driver version ZZZZ)
+May 08 12:14:41 myserver falco[15442]: Falco initialized with configuration file /etc/falco/falco.yaml
+May 08 12:14:41 myserver falco[15442]: Loading rules from file /etc/falco/falco_rules.yaml:
+May 08 12:14:42 myserver falco[15442]: Loading rules from file /etc/falco/falco_rules.local.yaml:
+May 08 12:14:42 myserver falco[15442]: Loading rules from file /etc/falco/k8s_audit_rules.yaml:
+May 08 12:14:43 myserver falco[15442]: Starting internal webserver, listening on port 8765
+May 08 12:14:43 myserver falco[15442]: gRPC server threadiness equals to 2
+May 08 12:14:43 myserver falco[15442]: grpc: {"created":"@1652026483.513407096","description":"No address added out of total 1 resolved","file":"/build/release/grpc-prefix/src/grpc/src/core/e>
+May 08 12:14:43 myserver falco[15442]: Error starting gRPC server
+```
+
+You may want to run SELinux in 'permissive' mode while you figure out how to give Falco the proper permission to create the gRPC server on startup:
+
+```shell
+sudo setenforce 0
+```
+
+To make this permanent across reboots (if you have Grubby installed):
```

Comment:
Hello,

My steps are to let SELinux run on "Permissive" mode, which will allow the application run but it will log what SELinux considers is an inappropriate behavior; Troubleshooting SELinux is a time consuming issue and sadly I don't have enough cycles right now on how to write a step to step tutorial how to add the rules to make it work (in reality I haven't seen the first large company that uses SELinux properly, that's why Falco is so  attractive).

Maybe for another update on the documentation? I can change the language explaining than using the permissive mode will allow the user to explore the issue and works toward a solution if SELinux is a must.

If not, let me know and I will retract this pull request.

Thanks.

Reply:
Thanks Jose. First of all I agree with you that the change to the documentation is valuable, so no need to retract the PR :rocket: ! The error message is confusing and knowing that it may be a SELinux issue is a great thing to have in the manual, thanks a lot for documenting it. It also makes sense that adding something more detailed can be done in another PR if you, or any other SELinux expert, has time to write it.

Here's what I have in mind: instead of adding the code snippet to disable SELinux permanently, have the pointers to the official documentation so users are well informed about what they're doing. I'll send you a suggestion snippet to show you what I mean more precisely.